benchmark/torch/ddpg/train.py

```diff
@@ -0,0 +1,157 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import random
+import torch
+import gym
+import argparse
+from parl.utils import logger, tensorboard, ReplayMemory
+from mujoco_model import MujocoModel
+from mujoco_agent import MujocoAgent
+from parl.algorithms import DDPG
+
+
+# Runs policy for 5 episodes by default and returns average reward
+# A fixed seed is used for the eval environment
+def eval_policy(agent, env_name, seed, eval_episodes=5):
+    eval_env = gym.make(env_name)
+    eval_env.seed(seed + 100)
+
+    avg_reward = 0.
+    for _ in range(eval_episodes):
+        state, done = eval_env.reset(), False
+        while not done:
+            action = agent.predict(np.array(state))
+            state, reward, done, _ = eval_env.step(action)
+            avg_reward += reward
+    avg_reward /= eval_episodes
+    return avg_reward
+
+
+if __name__ == "__main__":
```

Comment:
please split this part to several functions

Reply:
Resolved. Have split it to three defs(main, run_train_episode, run_evaluate_episodes)
Also rerun it on GPU server, the performance has not decreased and even much better for seed 0.

![image](https://user-images.githubusercontent.com/76139596/102681179-462e4900-41fa-11eb-8728-01a582dd7ca9.png)
![image](https://user-images.githubusercontent.com/76139596/102681190-59411900-41fa-11eb-9c71-d212d63e7641.png)
![image](https://user-images.githubusercontent.com/76139596/102681196-69f18f00-41fa-11eb-93cc-269439107e02.png)